.github/workflows/hyperliquid-node.yml

```
@@ -0,0 +1,115 @@
+      - name: Git Checkout (Hyperliquid)
+        uses: actions/checkout@v4
+        with:
+          repository: 'hyperliquid-dex/node'
+          ref: 'main'
+          path: 'hyperliquid-node'
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)